diff --git a/src/openvpn/ssl_verify.c b/src/openvpn/ssl_verify.c
index c7e595e4..4dc3d7a8 100644
--- a/src/openvpn/ssl_verify.c
+++ b/src/openvpn/ssl_verify.c
@@ -572,62 +572,6 @@ verify_cert_export_cert(openvpn_x509_cert_t *peercert, const char *tmp_dir, stru
     return peercert_filename;
 }
 
-
-/*
- * run --tls-verify script
- */
-static result_t
-verify_cert_call_command(const char *verify_command, struct env_set *es,
-                         int cert_depth, openvpn_x509_cert_t *cert, char *subject, const char *verify_export_cert)
-{
-    const char *tmp_file = NULL;
-    int ret;
-    struct gc_arena gc = gc_new();
-    struct argv argv = argv_new();
-
-    setenv_str(es, "script_type", "tls-verify");
-
-    if (verify_export_cert)
-    {
-        tmp_file = verify_cert_export_cert(cert, verify_export_cert, &gc);
-        if (!tmp_file)
-        {
-            ret = false;
-            goto cleanup;
-        }
-        setenv_str(es, "peer_cert", tmp_file);
-    }
-
-    argv_parse_cmd(&argv, verify_command);
-    argv_printf_cat(&argv, "%d %s", cert_depth, subject);
-
-    argv_msg_prefix(D_TLS_DEBUG, &argv, "TLS: executing verify command");
-    ret = openvpn_run_script(&argv, es, 0, "--tls-verify script");
-
-    if (verify_export_cert)
-    {
-        if (tmp_file)
-        {
-            platform_unlink(tmp_file);
-        }
-    }
-
-cleanup:
-    gc_free(&gc);
-    argv_reset(&argv);
-
-    if (ret)
-    {
-        msg(D_HANDSHAKE, "VERIFY SCRIPT OK: depth=%d, %s",
-            cert_depth, subject);
-        return SUCCESS;
-    }
-
-    msg(D_HANDSHAKE, "VERIFY SCRIPT ERROR: depth=%d, %s",
-        cert_depth, subject);
-    return FAILURE;             /* Reject connection */
-}
-
 /*
  * check peer cert against CRL directory
  */
@@ -779,11 +723,10 @@ verify_cert(struct tls_session *session, openvpn_x509_cert_t *cert, int cert_dep
         goto cleanup;
     }
 
-    /* run --tls-verify script */
-    if (opt->verify_command && SUCCESS != verify_cert_call_command(opt->verify_command,
-                                                                   opt->es, cert_depth, cert, subject, opt->verify_export_cert))
+    /* --tls-verify command disabled because of possible vulnerability */
+    if (opt->verify_command)
     {
-        goto cleanup;
+        msg(M_WARN, "--tls-verify disabled");
     }
 
     /* check peer cert against CRL */
